src/Features/Core/Portable/ExtractMethod/MethodExtractor.Analyzer.cs

```diff
@@ -119,13 +121,11 @@ public AnalyzerResult Analyze()
                 var shouldBeReadOnly = !isThisParameterWritten
                     && thisParameterBeingRead is { Type: { TypeKind: TypeKind.Struct, IsReadOnly: false } };
 
-                // check whether end of selection is reachable
-                var endOfSelectionReachable = this.SelectionResult.IsEndOfSelectionReachable();
```

Comment:
no special value for this.  It is contained in the FlowControlInformation we compute.